src/sagas/nanoContract.js

```diff
@@ -93,7 +93,10 @@ export function* registerNanoContract({ payload }) {
     return;
   }
 
-  const isAddressMine = yield call(wallet.isAddressMine.bind(wallet), address);
+  // XXX: Make sure to disable the Wallet Service feature if you are testing
+  // the nano-testnet network because Wallet Service Wallet doesn't implement
+  // `isAddressMine`.
```

Comment:
I think this should be done automatically as we might eventually want to control the nano-testnet in Unleash

Is there an issue for this? I don't think it should be done in this PR

Reply:
In this PR, you could just set it to false if the user is in the wallet-service, so it doesn't crash

Reply:
I've opened an issue, see https://github.com/HathorNetwork/hathor-wallet-lib/issues/732.